doc/JSON_FLAGS.md

```diff
@@ -588,6 +588,7 @@ List of known flags, used in both `terrain.json` and `furniture.json`.
 - ```BUTCHER_EQ``` Butcher's equipment - required for full butchery of corpses.
 - ```CAN_SIT``` Furniture the player can sit on. Player sitting near furniture with the `FLAT_SURF` tag will get mood bonus for eating.
 - ```CHIP``` Used in construction menu to determine if wall can have paint chipped off.
+- ```CLEAN_WATER``` Liquid taken from tiles with this flag is safe for drinking. 
```

Comment:
The code does something different. It just makes the liquid on tiles with this flag cold. Have you tried spawning dirty water instead `"liquids": { "W": { "liquid": "water", "amount": [ 0, 80 ] } }` - did it convert that water to clean water?

Also: what is the result of adding `TFLAG_MURKY` and `TFLAG_CLEAN_WATER` to the same terrain. According to the code (MURKY is checked first), it would make clean water have parasites.

Reply:
You're right. I removed the new flag altogether, it's useless.

> According to the code (MURKY is checked first), it would make clean water have parasites.

Yes, it will add parasites to any liquid (not only clean water).